doc/release-management.md

```diff
@@ -1,25 +1,18 @@
 # Release management
 
 1. Update CHANGELOG.md file with the list of changes and link
-2. Generate Operator yaml file with version
+2. Run `make release` to generate Operator manifest file and build
+   Containers for a specific version(and latest). Built Containers
+   will be published to dockerhub.
 
-        KADALU_VERSION=0.1.0 make gen-manifest
-        cp manifests/kadalu-operator.yaml manifests/kadalu-operator-0.1.0.yaml
-        make gen-manifest
+        KADALU_VERSION=0.1.0 make release
 
-3. Build Containers with the required version
+   **Note**: If `DOCKER_USER` is specified then it will publish the built
+   Containers to respective Dockerhub account.
 
-        KADALU_VERSION=0.1.0 make build-containers
-        docker push kadalu/kadalu-operator:0.1.0
-        docker push kadalu/kadalu-csi:0.1.0
-        docker push kadalu/kadalu-server:0.1.0
-        make build-containers
-        docker push kadalu/kadalu-operator:latest
-        docker push kadalu/kadalu-csi:latest
-        docker push kadalu/kadalu-server:latest
+        DOCKER_USER=aravindavk KADALU_VERSION=0.1.0 make release
 
-4. Send the PR with manifest file changes
-5. Tag and Release with release notes. Operator link will be(Short URL will be planned soon)
+3. Send the PR with manifest file and CHANGELOG.md file changes
+4. Tag and Release with release notes. Operator link will be(Short URL will be planned soon)
```

Comment:
This is incomplete sentence at the time. Can this be changed?

Reply:
Updated.